backend/accounts/models.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class CustomUser(AbstractUser):
+    USERNAME_FIELD = EMAIL_FIELD = "email"
+    REQUIRED_FIELDS = []
+
+    email = models.EmailField(_("email address"), unique=True)
+
+    class Meta:
+        db_table = "auth_user"
```

Comment:
To je tady proc? Za normalnich okolnosti neni treba Djangu do generovanych jmen DB tabulek kecat.

Reply:
(pokud to das pryc, pregeneruj prosim i migrace, protoze ted se v nich tenhle konstrukt objevuje taky)